web/src/translation/translation.ts

```diff
@@ -1,11 +1,10 @@
-import { TFunction } from 'i18next';
 import { useTranslation as useTranslationHook } from 'react-i18next';
 import { vsprintf } from 'sprintf-js';
 import i18next from './i18n';
 
 // Todo: We should get rid of vsprintf and use i18next interpolation instead
 const translateWithTranslator = (
-  translator: TFunction,
```

Comment:
This caused:
```
src/translation/translation.ts:1:10 - error TS2614: Module '"i18next"' has no exported member 'TFunction'. Did you mean to use 'import TFunction from "i18next"' instead?

1 import { TFunction } from 'i18next';
```
We could also import t directly from i18next if you prefer, the type should be the same.

Reply:
Might be a tinsy bit nicer to import the type, but up to you :)

Reply:
t has a type defined internally as:
```ts
t: TFunction<[DefaultNamespace, ...Exclude<FlatNamespace, DefaultNamespace>[]]>;
```
But TFunction is not exported so this way felt a little simpler without having to do any workaround or importing directly from their internal type files.